ArcGISRuntimeSDKQt_CppSamples/Maps/setMaxExtent/SetMaxExtent.cpp

```diff
@@ -0,0 +1,89 @@
+// [WriteFile Name=SetMaxExtent, Category=Maps]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+#include "SetMaxExtent.h"
+
+#include "Map.h"
+#include "MapQuickView.h"
+#include "Graphic.h"
+#include "GraphicsOverlay.h"
+#include "SimpleLineSymbol.h"
```

Comment:
Same comment as in header file: Our samples are inconsistent in this, but what do you think about alphabetizing this list?